stdlib/ocaml/generate.mc

```diff
@@ -402,8 +383,42 @@ lang OCamlGenerate = MExprAst + OCamlAst + OCamlMatchGenerate
       info = NoInfo ()
     }
   -- TmExt Generation
-  | TmExt _ ->
-    error "externals expected to be generated in a previous step"
+  | TmExt {ident = ident, ty = ty, inexpr = inexpr, info = info} ->
+    match mapLookup ident env.exts with Some r then
+      let r : ExternalImpl = head r in
+      match convertData info env (OTmVarExt { ident = r.ident }) r.ty ty
+      with (_, body) then
+        -- let names = create (typeArity ty) (lam. nameSym "x") in
+        -- let vars = map (lam name. TmVar { ident = name, info = info }) names in
+        -- let body =
+        --   foldl (tmApp info (TyUnknown { info = info })) (body) vars
+        -- in
+        -- let body =
+        --   foldr
+        --     (lam name. lam body.
+        --       TmLam {
+        --         ident = name,
+        --         tyIdent = TyUnknown { info = info },
+        --         ty = TyUnknown { info = info },
+        --         body = body,
+        --         info = info
+        --       })
+        --     body
+        --     names
+        -- in
+        -- let body = generate env body in
```

Comment:
Perhaps this commented-out code should be removed? Or un-commented?

Reply:
Yes, thanks!